originals/a.yaml

```diff
@@ -115,6 +115,22 @@
     genre:
     - Action
 
+- name: Alan Wake
```

Comment:
looks like this and aladdin aren't sorted properly (should be above albion)
could you please move them up? thanks

Reply:
Sure, fixed. Please note that `games/o.yaml` is also wrongly sorted, because humans are often unsure about the sort order of letters, spaces and dashes (myself included). Perhaps a sorting script would be a good idea ;-)